web/src/routing/routes/UsersAdministration/Users/UserDetails/UserDetails.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState, VFC } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+
+import { Box, Typography } from "@mui/material";
+
+import { UsersAdministrationLayout } from "@layouts/UsersAdministrationLayout/UsersAdministrationLayout";
+import { ListElement } from "@components/ListElement/ListElement";
+import { Button } from "@components/Button/Button";
+import { ConfirmationModal } from "@components/ConfirmationModal/ConfirmationModal";
+import { rgbaColors } from "@themes/palette";
+import { EditUserModal } from "../Modals/EditUserModal/EditUserModal";
+import { AddToGroupModal } from "@routing/routes/UsersAdministration/Groups/Modals/AddToGroupModal/AddToGroupModal";
+import { User } from "@janush-types/user";
+
+interface LocationState {
+  user: User;
+}
+
+const UserDetails: VFC = () => {
+  const navigate = useNavigate();
+
+  const location = useLocation();
+  // TODO: change to get user data from backend
+  const { user } = location.state as LocationState;
+
+  const [showEditUserModal, setShowEditUserModal] = useState(false);
+  const [showAddToGroupModal, setShowAddToGroupModal] = useState(false);
+  const [showRemovingUserModal, setShowRemovingUserModal] = useState(false);
+
+  return (
+    <UsersAdministrationLayout
+      onBackClick={() => navigate(-1)}
+      buttons={
+        <>
+          <Button sx={{ mr: 2 }} onClick={() => setShowEditUserModal(true)}>
```

Comment:
Can you set `sx={{ mr: 2 }}` to the wrapper instead of placing in on every component?